source/components/auth/auth.js

```diff
@@ -17,7 +17,8 @@ class Auth extends Component {
     .then(
       (token) => {
         sessionStorage.setItem("Authorization", 'Bearer ' + token.data.data.auth);
-        this.props.history.push("/register");
+        const callback = sessionStorage.getItem('callback');

```

Comment:
We should have a "default" callback in case it's not set in sessionStorage already. (Probably `/register`)

Reply:
going to /start or /register will automatically place it into sessionStorage in componentWillMount before the redirect occurs in DidMount

Reply:
yeah but what if someone goes to `/auth`?

Reply:
I am not a smart man